doc/tutorials/imgproc/imgtrans/distance_transformation/distance_transform.markdown

```diff
@@ -0,0 +1,163 @@
+Image Segmentation with Distance Transform and Watershed Algorithm {#distance_transform}
+=============
+
+Goal
+----
+
+In this tutorial you will learn how to:
+
+-   Use the OpenCV function @ref cv::filter2D in order to perform some laplacian filtering for image sharpening
+-   Use the OpenCV function @ref cv::distanceTransform in order to obtain the derived representation of a binary image, where the value of each pixel is replaced by its distance to the nearest background pixel
+-   Use the OpenCV function @ref cv::watershed in order to isolate objects in the image from the background
+
+Theory
+------
+
+Code
+----
+
+This tutorial code's is shown lines below. You can also download it from
+    [here](https://github.com/Itseez/opencv/tree/master/samples/cpp/tutorial_code/ImgTrans/image_segmentation.cpp).
+@includelineno samples/cpp/tutorial_code/ImgTrans/image_segmentation.cpp
```

Comment:
file name is wrong here, should be `samples/cpp/tutorial_code/ImgTrans/imageSegmentation.cpp`